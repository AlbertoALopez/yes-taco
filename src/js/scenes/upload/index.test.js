import React from 'react';
import Upload from './index.js';

test('Dialog renders correctly', () => {
  const wrapper = shallow(<Upload />);

  expect(wrapper).toMatchSnapshot();
});
