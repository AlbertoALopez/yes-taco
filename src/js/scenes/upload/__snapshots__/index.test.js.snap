// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog renders correctly 1`] = `
<Dialog
  dialogText="This is where you upload"
  transitionId=""
/>
`;
